src/splitInteger.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger()).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(5, 1)).toStrictEqual([5]);
+  });
+
```

Comment:
Are you aware about this requirement - "there are parts elements (for example, if parts = 5, the num would be splitted into 5 integers);" ?

Reply:
You are right